cluster-autoscaler/cloudprovider/gce/gce_price_model.go

```diff
@@ -31,15 +32,53 @@ type GcePriceModel struct {
 
 const (
 	//TODO: Move it to a config file.
-	cpuPricePerHour         = 0.033174
-	memoryPricePerHourPerGb = 0.004446
-	preemptibleDiscount     = 0.00698 / 0.033174
+	cpuPricePerHour         = 0.022890
+	memoryPricePerHourPerGb = 0.003067
+	preemptibleDiscount     = 0.006867 / 0.022890
 	gpuPricePerHour         = 0.700
 
 	preemptibleLabel = "cloud.google.com/gke-preemptible"
 )
 
 var (
+	predefinedCpuPricePerHour = map[string]float64{
+		"e2":  0.021811,
```

Comment:
Yeah, adding them to predefined cost values makes a lot of sense.
Not necessarily to the custom costs, but this will still handle new predefined instance types in these families well.

Reply:
https://cloud.google.com/compute/vm-instance-pricing
Defines the cpu/memory per hour pricing for C2, E1, M1, N1, N2 and N2D, so they will have to suffice.